topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Inspecting the general information of `EMTABesethealthy.expression.tabular` we can see that there are **1,097** cells in the data as there are 1,098 columns (we need to subtract 1 for the header).
+> > 2. Looking at the output of the {% tool [Count](Count1) %} tool (or the above table), there are **14** distinct cell types in the data.
+> >
+> {: .solution}
+>
+{: .question}
+
+# Process the single-cell data
+
+In order to get a good understanding of the accuracy of our deconvolution tools, we are going to run our evaluations multiple times. This approach ensures that a single good or bad evaluation does not disproportionately represent the tool's overall performance.
+
+However, instead of running all of our tools multiple times for each evaluation (which would be quite time consuming!), we will leverage "batch computation" in Galaxy. By storing our data in collections, any tools or workflows that use those collections will automatically run multiple times (once for each element in the collection). We will now perform some pre-processing of our data to get it into the right format.
+
+## Transpose expression matrix
+
+If we inspect the expression data file downloaded earlier, we can see that currently the rows represent genes and columns represent cells. However, this needs to be swapped for the later workflows. To fix this we will transpose the expression matrix.
+
+> <hands-on-title>Transpose expression matrix</hands-on-title>
+>
+> 1. {% tool [Transpose](toolshed.g2.bx.psu.edu/repos/iuc/datamash_transpose/datamash_transpose/1.8+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input tabular dataset"*: `EMTABesethealthy.expression.tabular`
+>
+> 2. **Rename** {% icon galaxy-pencil %} output `Transposed expression matrix`
+>
+{: .hands_on}
+
+## Generate batch mode collections
+
+For this tutorial we will run the evaluations **20** times, this will both help improve the sample size and allow us to determine the consistency of the tools, whilst being small enough to run in a reasonable amount of time.
+
+We will now duplicate our single-cell data 20 times and store it in a collection. This will be done for both the expression data and metadata files.
+
+> <hands-on-title>Generate collections from data</hands-on-title>
+>
+> 1. {% tool [Duplicate file to collection](__DUPLICATE_FILE_TO_COLLECTION__) %} with the following parameters:
+>    - {% icon param-file %} *"Input Dataset"*: `EMTABesethealthy.phenotype.tabular`
+>    - *"Size of output collection"*: `20`
+>
+> 2. **Rename** {% icon galaxy-pencil %} output `Metadata`
+>
+> 3. {% tool [Duplicate file to collection](__DUPLICATE_FILE_TO_COLLECTION__) %} with the following parameters:
+>    - {% icon param-file %} *"Input Dataset"*: `Transposed expression matrix`
+>    - *"Size of output collection"*: `20`
+>
+> 4. **Rename** {% icon galaxy-pencil %} output `Expression data`

```

Comment:
I would recommend thinking about keeping the terms consistent, as later on when the first workflow is run, these inputs are described with different terminology